core/executor/wasm/build.sh

```diff
@@ -6,7 +6,7 @@ if cargo --version | grep -q "nightly"; then
 else
 	CARGO_CMD="cargo +nightly"
 fi
-CARGO_INCREMENTAL=0 RUSTFLAGS="-C link-arg=--export-table" $CARGO_CMD build --target=wasm32-unknown-unknown --release
+CARGO_INCREMENTAL=0 RUSTFLAGS="-C link-arg=--export-table" $CARGO_CMD build --target=wasm32-unknown-unknown --release $CARGO_BUILD_EXTRA_OPTIONS
```

Comment:
haven't tested but wouldn't you need to append `$@` instead of `$CARGO_BUILD_EXTRA_OPTIONS` when you write further down in `./scripts/build.sh`:
```
./build.sh $CARGO_BUILD_EXTRA_OPTIONS
```
otherwise ^^ one is unneded if the env variable should be taken directly from the `.gitlab-ci.yml`.

Reply:
Well, it's working, so I don't know.